NitroxTest/Serialization/WorldPersistenceTest.cs

```diff
@@ -301,6 +301,7 @@ public void PlayerDataTest()
                     Assert.AreEqual(playerData.Permissions, playerDataAfter.Permissions, "PlayerData.Players.Permissions is not equal");
                     Assert.AreEqual(playerData.NitroxId, playerDataAfter.NitroxId, "PlayerData.Players.NitroxId is not equal");
                     Assert.AreEqual(playerData.IsPermaDeath, playerDataAfter.IsPermaDeath, "PlayerData.Players.IsFurniture is not equal");
+                    Assert.AreEqual(playerData.Token, playerDataAfter.Token, "PlayerData.Players.Token is not equal");
```

Comment:
In addition to that you need generate a new token in `GeneratePersistedWorldData()`

Reply:
I have added the requested changes in a subsequent commit. Please confirm.